### DISCUSSION TITLE
504 (Outdated Optimize Dep) when using primereact/datatable

### DISCUSSION THREAD

**Discussion by mukeshsjpl** [primefaces/community#302] [PrimeReact]
Hello all,
I am using  react js with Vite,

Now I am facing problem while using primereact/datatable and error is givven below,

GET http://localhost:5173/node_modules/.vite/deps/primereact_column.js?v=16f7fa0a net::ERR_ABORTED 504 (Outdated Optimize Dep)

Please help me out.
thanks
Mukesh Mishra

**Comment by melloware** [comment]
I use Vite and I have no issues?

**Reply by mukeshsjpl** [reply]
its automatically solve when I restarted my laptop, Thanks for your support

**Comment by mukeshsjpl** [comment]
its automatically solve when I restarted my laptop, Thanks for your support
